babel/babel
# Refactor trailing comment adjustment

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes issues similar to 10368
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

Following up from https://github.com/babel/babel/pull/10369

- Unify the logic for adjusting trailing comments into a separate function
- Use it for all three cases, which fixes a bug for ObjectExpressions and CallExpressions
- Update tests to check for the fixed bug
